docs/topics/Gateway.md

```diff
@@ -79,7 +79,7 @@ def on_websocket_message(msg):
 
   # if the message *does* end with ZLIB_SUFFIX,
   # get the full message by decompressing the buffers
-  msg = inflator.decompress(buffer).decode('utf-8')
+  msg = inflator.decompress(buffer)
```

Comment:
I believe this was added to imply that you can expect UTF-8 or any unicode variant. It is a somewhat awkward way to show this (I know that json.loads in Python consumes `bytes` as well as `str`), but so this detail isn't lost, it is probably worth adding a footnote that points this out somewhere near here.